rio/test/RIO/LoggerSpec.hs

```diff
@@ -24,3 +24,13 @@ spec = do
       logStickyDone "XYZ"
     builder <- readIORef ref
     toLazyByteString builder `shouldBe` "ABC\b\b\b   \b\b\bshould appear\nABC\b\b\b   \b\b\bXYZ\n"
+  it "updateLogMinLevel" $ do
+    (ref, options) <- logOptionsMemory
+    withLogFunc (options & setLogMinLevel LevelDebug)
+      $ \lf -> runRIO lf $ do
+        logDebug "should appear"
+        -- reset log min level to info
+        updateLogMinLevel lf LevelInfo
```

Comment:
Note: This test is not as realistic, the `updateLogMinLevel` function will be likely called from a different environment than the logging environment.